app/controllers/api/service_providers_controller.rb

```diff
@@ -25,7 +25,7 @@ def serialized_service_providers(service_providers)
     end
 
     def approved_service_providers
-      ServiceProvider.includes(:logo_file_attachment, :agency).all
+      ServiceProvider.includes(logo_file_attachment: :blob).includes(:agency).all
```

Comment:
I *think* this syntax works too
```suggestion
      ServiceProvider.includes(:agency, logo_file_attachment: :blob).all
```

Reply:
Confirmed!

Reply:
Ahhhh, that's the ticket. I tried `ServiceProvider.includes(logo_file_attachment: :blob, :agency)` (which doesn't work because the syntax is not valid now that I look again)